enterprise/server/remote_execution/dirtools/dirtools.go

```diff
@@ -357,6 +355,32 @@ func UploadTree(ctx context.Context, env environment.Env, dirHelper *DirHelper,
 				if err != nil {
```

Comment:
I was considering adding it here 

```
				if !(dirHelper.ShouldUploadFile(fqfn) || dirHelper.IsOutputPath(fqfn)) {
					continue
				}
```

But I was worried about breaking existing behavior.

It's unclear to me how often people create/include symlink outputs in their actions _without_ declaring them in `Command.output_path`, and if we should start explicitly requiring folks to declare them.